performance-tests/perf-test-commits.sh

```diff
@@ -0,0 +1,113 @@
+#! /usr/bin/env bash
```

Comment:
I'm not sure if it is significant, but the space looks odd.

```suggestion
#!/usr/bin/env bash
```